service/src/intTest/java/uk/nhs/adaptors/gp2gp/MessageQueueTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.nhs.adaptors.gp2gp;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jms.core.JmsTemplate;
+
+import uk.nhs.adaptors.gp2gp.extension.ActiveMQExtension;
+import uk.nhs.adaptors.gp2gp.extension.MongoDBExtension;
+
+@SpringBootTest
+@ExtendWith({MongoDBExtension.class, ActiveMQExtension.class})
+public class MessageQueueTest {
+    private static final String INVALID_MESSAGE_CONTENT = "TRASH";
+    private static final String DLQ = "ActiveMQ.DLQ";
```

Comment:
Would this work against AmazonMQ or Azure Service Bus? I'll give you a hand with the testing for this.